eventdata/challenges/daily-log-volume-index-and-query.json

```diff
@@ -41,10 +41,21 @@
         "index": "elasticlogs-2999-01-01-throughput-test"
       }
     },
+    {
+      "name": "check-cluster-health",
+      "operation": {
```

Comment:
we should add a `"retry-until-success": true` here (similar to https://github.com/elastic/rally-eventdata-track/pull/97).